time/checktime.go

```diff
@@ -198,6 +198,7 @@ func (client *TimeClient) Reconfigure(config *ClientConfig) (err error) {
 
 	if !config.NoTLS {
 		client.client = &http.Client{
+			Timeout: 35 * time.Second,
```

Comment:
Any particular reason for '35' seconds?

Reply:
no reason for exact `35` sec, should be more then all other timeouts and KeepAlive which is 30 sec